src/component/pomodora.js

```diff
@@ -0,0 +1,57 @@
+import "../styles.css";
+
+export default function Pomodora() {
+  return (
+    <div className="pomodora-container">
+      <div className="pomodora-clock-container">
+        <div className="pomodora-header">
+          <h3>Pomodora Clock</h3>
+        </div>
+        <div className="times">
+          <div className="times-content">
+            <span id="time-left">45:16</span>
+            <label className="timer-total text-grey">out of 60min</label>
+          </div>
+        </div>
+        <div className="pomodora-btn">
+          <button className="button button-primary bold ptr">Start</button>
+          <button className="button button-secondary bold ptr">Reset</button>
+        </div>
+      </div>
+      <div className="note-detail-container card">
+        <div className="note-container">
+          <div className="note-header">
+            <h4>Note Title...</h4>
+          </div>
+          <div className="note-body">
+            <p className="bold p2 text-grey">
+              Lorem Ipsum es simplemente el texto de relleno de las imprentas y
```

Comment:
Suggestion: Can keep dummy text in separate file, like headerContent.js, to keep logic and content separate and easily manageble